src/Event/OrganisationUserIdEvent.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\os2forms_organisation\Event;
+
+use Drupal\Component\EventDispatcher\Event;
+
+/**
+ * Organisation user id event.
+ */
+class OrganisationUserIdEvent extends Event {
+  const EVENT_NAME = 'organisation_user_id_event';
+
+  /**
+   * The id.
+   *
+   * @var string
+   */
+  private string $id;
+
+  /**
+   * The constructor.
+   */
+  public function __construct() {
+    $this->id = '';
```

Comment:
Why not `null`? And why not just initialize the id when it is declared?

Reply:
Fair point. Updated.